sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1558,6 +1566,25 @@ val GPU_COREDUMP_PIPE_PATTERN = conf("spark.rapids.gpu.coreDump.pipePattern")
     .checkValue(v => v >= 0 && v <= 1, "The ratio value must be in [0, 1].")
     .createWithDefault(1.0)
 
+  val AGG_OUTPUT_SIZE_RATIO = conf("spark.rapids.sql.agg.outputSizeRatioToBatchSize")
+    .doc("The ratio of the output size of an aggregation to the batch size. ")
+    .internal()
+    .doubleConf
+    .checkValue(v => v > 0 , "The ratio value must be greater than 0.")
+    .createWithDefault(1.0)
+
+  val FALLBACK_ALGORITHM_FOR_OVERSIZE_AGG = conf("spark.rapids.sql.agg.fallbackAlgorithm")
+    .doc("When agg cannot be done in a single pass, use sort-based fallback or " +
+      "repartition-based fallback.")
+    .stringConf
+    .checkValues(Set("sort", "repartition"))
+    .createWithDefault("repartition")
+
+  // todo: remove this
```

Comment:
Do we have an follow on issue for that.

Reply:
This piece of code is irrelavant to repartition based agg, so it will be removed by a subsequent commit